cmake/projects/zookeeper/hunter.cmake

```diff
@@ -0,0 +1,27 @@
+# cmake/projects/Example/hunter.cmake
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cacheable)
+
+# List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    zookeeper
+    VERSION
+    "3.4.9-p1"
```

Comment:
I figure I should only increment the hunter package version for releases that actually got released, whereas everything so far has just been internal testing releases.
I can delete the old releases though and make a new one that matches the name in the package file.